nhost/compose/hostnames.go

```diff
@@ -0,0 +1,84 @@
+package compose
+
+import (
+	"fmt"
+)
+
+const (
+	// backend service hosts
+	hostLocalhost              = "localhost"
+	HostLocalDashboardNhostRun = "local.nhost.run"
+	HostLocalGraphqlNhostRun   = "local.graphql.nhost.run"
+	hostHasuraConsole          = hostLocalhost
+	hostMinio                  = hostLocalhost
+	HostLocalAuthNhostRun      = "local.auth.nhost.run"
+	HostLocalStorageNhostRun   = "local.storage.nhost.run"
+	HostLocalFunctionsNhostRun = "local.functions.nhost.run"
+	HostLocalMailNhostRun      = "local.mail.nhost.run"
+)
+
+func DashboardHostname(port uint32) string {
+	return sslHostnameWithPort(HostLocalDashboardNhostRun, port)
+}
+
+func HasuraGraphqlHostname(port uint32) string {
+	return sslHostnameWithPort(HostLocalGraphqlNhostRun, port)
+}
+
+func MinioHostname(port uint32) string {
+	return httpHostnameWithPort(hostMinio, port)
+}
+
+func HasuraConsoleHostname(port uint32) string {
+	return httpHostnameWithPort(hostHasuraConsole, port)
+}
+
+func HasuraAPIHostname(port uint32) string {
```

Comment:
do you mean hasura metadata api here?

Reply:
Actually I realized this method isn't needed anymore, updated the code